app/src/main/res/values/strings.xml

```diff
@@ -868,9 +868,9 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="team_mode">Team Mode</string>
     <string name="team_mode_exit">Exit Team Mode</string>
     <string name="team_mode_description">When mapping the same area together in a group, you can enter team mode to split up quests between all team members.</string>
-    <string name="team_mode_team_size_label">How many people are there?</string>
+    <string name="team_mode_team_size_label2">How many people are mapping? Enter the same number on everyone\'s phone.</string>
     <string name="team_mode_team_size_hint">Team mode works for groups of 2 to 12 people.</string>
-    <string name="team_mode_choose_color2">Now everybody in your group chooses a unique color on their phone:</string>
+    <string name="team_mode_choose_color2">Next, each person must pick a different color. Choose yours:</string>
```

Comment:
or this?
```suggestion
    <string name="team_mode_choose_color2">Next, each person picks a different color. Choose yours:</string>
```

Reply:
I don't think so. This string would need to be a lot shorter to fit on one line, and it works fine on two lines, so I don't see a need.

The phrasing is slightly worse for this context. I would expect a present-tense verb like this when describing the process **to a third party**. "First, everyone enters the number of people in the group. Then, each person picks a unique color." When presented **as instructions**, it comes across a little stilted, as if it's already in progress, when really it's something that's about to happen. It ends up being clear-*enough*, just not native-sounding.